tests/Unit/LogisticsOrderFactoryTest.php

```diff
@@ -82,35 +82,42 @@ public function setUp(): void
         ];
     }
 
-    public function test_it_adds_the_callback_url_to_the_logistics_order()
+    public function test_it_adds_the_success_callback_url_to_the_payload()
     {
         $logisticsOrder = $this->logisticsOrderFactory->buildFromWooCommerceOrder($this->createMockOrder());
 
-        $this->assertSame("http://test.com/wp-json/gebrueder-weiss-woocommerce/v1/update/12", $logisticsOrder->getUrl());
+        $this->assertSame("http://test.com/wp-json/gebrueder-weiss-woocommerce/v1/orders/12/callbacks/success", $logisticsOrder->getCallbacks()->getSuccessCallback());
     }
 
-    public function test_it_adds_the_created_date_to_the_logistics_order()
+    public function test_it_adds_the_fulfillment_callback_url_to_the_payload()
     {
         $logisticsOrder = $this->logisticsOrderFactory->buildFromWooCommerceOrder($this->createMockOrder());
 
+        $this->assertSame("http://test.com/wp-json/gebrueder-weiss-woocommerce/v1/orders/12/callbacks/fulfillment", $logisticsOrder->getCallbacks()->getFullfilledCallback());
+    }
+
+    public function test_it_adds_the_created_date_to_the_logistics_order()
+    {
+        $logisticsOrder = $this->logisticsOrderFactory->buildFromWooCommerceOrder($this->createMockOrder())->getLogisticsOrder();
+
         $this->assertSame("2021-07-29T14:53:52+00:00", $logisticsOrder->getCreationDateTime()->format(DateTimeInterface::RFC3339));
     }
 
     public function test_it_adds_the_owner_id_to_the_logistics_order()
     {
-        $logisticsOrder = $this->logisticsOrderFactory->buildFromWooCommerceOrder($this->createMockOrder());
+        $logisticsOrder = $this->logisticsOrderFactory->buildFromWooCommerceOrder($this->createMockOrder())->getLogisticsOrder();
 
-        $this->assertSame(420000, $logisticsOrder->getOwnerId());
+        $this->assertSame(420000, $logisticsOrder->getCustomerId());
     }
 
     public function test_it_correctly_adds_the_consignee_address()
     {
-        $logisticsOrder = $this->logisticsOrderFactory->buildFromWooCommerceOrder($this->createMockOrder());
+        $logisticsOrder = $this->logisticsOrderFactory->buildFromWooCommerceOrder($this->createMockOrder())->getLogisticsOrder();
 
         $logisticsOrder->getLogisticsAddresses();
         $address = $logisticsOrder->getLogisticsAddresses()[0];
 
-        $this->assertSame("consignee", $address->getAddressType());
+        $this->assertSame("CONSIGNEE", $address->getAddressType());
         $this->assertSame("first-name", $address->getAddress()->getName1());
```

Comment:
thats an ugly method name, is it coming from wooCommerce?

Reply:
That method is a generated one from the api cllient.